### ISSUE TITLE
enums are not being exported on transpile

### ISSUE THREAD

**Issue by kevupton** [angular/tsickle#805]
This is a reference to this issue:
https://github.com/dherges/ng-packagr/issues/801

But basically enums inside a namespace are not being exported on transpile time. This causes the treeshaker to delete them when the code is bundled.

```typescript
export interface Test {

}

export namespace Test {
  export enum TestEnum {
    Hello='Hello',
    World='World'
  }
}
```

In the example above, the code is output as:
```typescript
var Test;
(function (Test) {
    /** @enum {string} */
    const TestEnum = {
        Hello: 'Hello',
        World: 'World',
    };
})(Test || (Test = {}));
```

**Comment by kevupton** [comment]
@mprobst Hey Martin, 
whats the ETA on this getting into a release? This is quite a desired fix right now. 
❤️

**Comment by mprobst** [comment]
I've released this in `0.29.0`.
